src/ui.ts

```diff
@@ -291,7 +283,7 @@ public nodes: Nodes;
       /**
        * Preloader does not exists on first rendering with presaved data
        */
-      if (this.nodes.imagePreloader) {
+      if (this.nodes.imagePreloader != undefined) {
```

Comment:
```suggestion
      if (this.nodes.imagePreloader !== undefined) {
```

Reply:
and in other places

Reply:
I did it on purpose to check for `null` and `undefined` possibility.

Reply:
its better to check for them explicitly. And write jsdoc about cases when it can be undefined and when null